code/src/java/pcgen/cdom/facet/analysis/ResultFacet.java

```diff
@@ -57,14 +57,19 @@ public Object getGlobalVariable(CharID id, String varName)
 
 	public Object getLocalVariable(CharID id, CDOMObject cdo, String varName)
 	{
+		String localScopeName = cdo.getLocalScopeName();
+		if(localScopeName == null){
```

Comment:
Well, first, this is not our code standards :) ... space after if, open curly brace on a new line.
Other than that, yes, it looks correct